lib/routes/gov/hangzhou/crawler.ts

```diff
@@ -0,0 +1,73 @@
+import logger from '@/utils/logger';
+
+export async function zjzwfwCrawler(item: any, browser: any): Promise<string> {
+    let page;
+    try {
+        page = await browser.newPage();
+        await page.setRequestInterception(true);
+        let response = '';
+        try {
+            const CHUNK_REGEX = /chunk-vendors\.[a-f0-9]+\.(js|css)/;
+
+            const loadedResources = { css: false, js: false };
+            page.on('request', (request) => {
+                request.continue({
+                    ...request.headers(),
+                    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36',
+                    'Accept-Language': 'en-US,en;q=0.9',
```

Comment:
Puppeteer requests do have this header set. Are there any issues that it has to be to set twice?